packages/zk-transfer-web/src/libs/txHelpers.ts

```diff
@@ -0,0 +1,82 @@
+import BN from "bn.js";
+import { Map } from "immutable";
+import { Demo, EthAddress, BM, SignerEth } from "./types";
+import { untilTxMined } from "demo-tx";
+import { runTransforms } from "demo-transform";
+import { deployed, constructMintPipeline } from "demo-aztec-lib";
+import { RINKEBY_ADMIN_ADDRESS, RINKEBY_ADMIN_PASSWORD } from "./constants";
+
+const mintPipeline = constructMintPipeline();
+
+export const mint = async (state: any) => {
+  return await runTransforms(mintPipeline, state);
+};
+
+type makeMintOptions = { bm: BM; tradeSymbol: string; amount: number };
```

Comment:
as described above `amount` should be a `BN`.
A better name for `makeMintOptions` and `makeMint` would be `doMintOptions` and `doMint`.